host-controller/src/main/java/org/jboss/as/host/controller/ServerInventoryImpl.java

```diff
@@ -229,10 +228,11 @@ private String createServerAuthToken(final String serverName) {
         // this method.
 
         // Create a new serverAuthToken
-        final byte[] tokenBytes = new byte[16]; // 16 for now but will be making independent.
-        new Random(new SecureRandom().nextLong()).nextBytes(tokenBytes);
-        String serverAuthToken = Base64.getEncoder().encodeToString(tokenBytes);
-        serverAuthToken = "SIK" + serverAuthToken.substring(3);
```

Comment:
Ignore my previous comment, I see now that "SIK" was renamed to "SAT" here.

Reply:
+1 it started as Server Inventory Token so I could work out where these tokens were flying around, then became Server Authentication Token - it is not strickly needed but when you see the token it is reassuring it is the intended one.